inject/src/main/java/io/micronaut/context/env/AbstractPropertySourceLoader.java

```diff
@@ -152,4 +157,22 @@ protected void processMap(Map<String, Object> finalMap, Map map, String prefix)
             }
         }
     }
+
+    /**
+     * Return logEnabled value.
+     *
+     * @return is log enabled
```

Comment:
```suggestion
     * @return is log enabled
     * @since 3.9.0
```

Same in `PropertySourcePropertyResolver`

Reply:
fixe